insights/combiners/rhsm_release.py

```diff
@@ -37,17 +37,16 @@ def __init__(self, rhsm_release, sm_release, rhui_release):
 
         self.minor = None
         """ int: Minor version of the release """
-
-        if rhsm_release is not None:
+        if rhsm_release is not None and rhsm_release.set:
             self.set = rhsm_release.set
             self.major = rhsm_release.major
             self.minor = rhsm_release.minor
 
-        elif sm_release is not None:
+        elif sm_release is not None and sm_release.set:
             self.set = sm_release.set
             self.major = sm_release.major
             self.minor = sm_release.minor
-        else:
+        elif rhui_release is not None and rhui_release.set:
```

Comment:
For these if/else checks,  even in a non-match case, should we still consider to assign a parser result to this combiner?  To let this combiner inherit values from a parser instead of using the default init value. 
In this way, further changes to parsers or unexpected parsed values could be covered better. 
hey @huali027 , what do you think?